www/shims/jquery.html.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+/**
+ * @param {FetcherIndex} index to use for URL resolution
+ * @param {Object} context e.g. jQuery, $.fn, etc
+ * @param {String} method name e.g. 'html'
+ * @returns {Function} the shim
+ */
+
+module.exports = function (index, context, method) {
+  var oldMethod = context[method];
+
+  var fixSrc = function (whole, value) {
+    var fixValue;
+    if (value && typeof value === 'string') {
+      fixValue = index.resolveLocalUrl(value);
+      whole = whole.replace(value, fixValue);
+    }
+    return whole;
+  };
+
+  var fixHtml = function (value) {
+    if (value && typeof value === 'string') {
+      value = value.replace(/<img[-_\s\=\"\'\.\/\w]* src="([-_\.\/\w]+)"/gmi, fixSrc);
+    }
```

Comment:
Yeah, looks like some how copy paste lost the ‘:', I just fixed it, thank you so much!
> On 11 Aug 2015, at 5:05 pm, Simon Dobie <notifications@github.com> wrote:
> 
>  <img src="http://www.work.com/hello/bye.jpg <http://www.work.com/hello/bye.jpg>" />